packages/react-components/src/components/VideoTile.tsx

```diff
@@ -333,19 +333,17 @@ export const VideoTile = (props: VideoTileProps): JSX.Element => {
         rootStyles,
         {
           background: theme.palette.neutralLighter,
-          borderRadius: theme.effects.roundedCorner4,
-          // To ensure that the video tile is focusable when there is a floating video tile
-          zIndex: 1
+          borderRadius: theme.effects.roundedCorner4
         },
         isSpeaking && {
-          '&::before': {
+          '&::after': {
             content: `''`,
             position: 'absolute',
-            zIndex: 1,
             border: `0.25rem solid ${theme.palette.themePrimary}`,
             borderRadius: theme.effects.roundedCorner4,
             width: '100%',
-            height: '100%'
+            height: '100%',
+            pointerEvents: 'none'
```

Comment:
will this pointer event block any interactions with keyboard if on mobile? or is this just for the border?

Reply:
This won't block mouse or keyboard interactions with the VideoTile. This will just prevent mouse interactions with the [after pseudo-element](https://developer.mozilla.org/en-US/docs/Web/CSS/::after). Needed to do this because it blocks the onHover events on the VideoTile.